### DISCUSSION TITLE
Reply to the comment section of the chat

### DISCUSSION THREAD

**Discussion by 1f1n** [python-telegram-bot/python-telegram-bot#4281] [Q&A] [answered]
What I'm trying to achieve

1. Post the message in the **channel**
2. Reply to the message in the channel but in the "Leave a comment" section 

What I'm doing:
Sending the message to the channel,  grabbing the message ID via `json()['result']['message_id']` then trying to reply like this
```
sendReplyMessageInChat = requests.get(f"https://api.telegram.org/bot{token}/sendMessage", params={
                    "chat_id": discussion_channel_id,
                    "text": "this is a reply in the chat to the main one",
                    "reply_to_message_id": message_id
                    
                })```
                
 But it isn't working, can anybody help?

**Comment by Poolitzer** [comment, marked as answer]
Hi this issue tracker is about using the python-telegram-bot library, not requests.

And in general posting the error and not "is not working" is really more helpful

**Comment by ramonreyes11** [comment]
This information is beneficial for those who need it. I hope you will make many more posts like this
[myAARPMedicare Health](https://www.myaarpmedicarehealth.com)
